src/Sensor/SensorGps.cpp

```diff
@@ -64,6 +64,13 @@ void SensorGps::handle() {
 
         _lastUpdateMicros = thisUpdateMicros;
     }
+
+    float value = _gps->getHorizontalAccEst() / MILIMETERS_IN_METERS;
+    if (value > 0.0001 && value < 1000.0) {
```

Comment:
Why not value > 0?

Reply:
Sometimes when GPS is still starting up, it will report an accuracy level of 0, so this is basically checking to ensure it's a real value that's low enough to be remotely usable.